packages/legend-application-studio/src/components/editor/edit-panel/service-editor/testable/ServiceTestDataEditor.tsx

```diff
@@ -426,6 +442,14 @@ export const ServiceTestDataEditor = observer(
                   </button>
                 </div>
               </div>
+              {!testData.connectionsTestData.length && (
```

Comment:
Could we replace the whole top panel with this when there are no connection test data. right now there is still the empty right side which might confuse the users